update_employees.py

```diff
@@ -416,9 +422,11 @@ def main():
     logging.info(f"{len(cleaned_payload)} total records to process in Knack.")
 
     result["errors"] = []
+
     for record in cleaned_payload:
         method = "update" if record.get("id") else "create"
         try:
+            logging.info(f"{method} {record[email_field]['email']}")
```

Comment:
while testing this PR, I ran into an error
```

  File "./update_employees.py", line 430, in main
    logging.info(f"{method} {record[email_field]['email']}")
KeyError: 'field_18'
```
and i found it was because one of the records in the payload only had an id and field_20: 'inactive'

Reply:
None of the records that will be marked as inactive have emails in the payload, we could add the email field though, so the logging would be consistent?

Reply:
Good idea: fixed 🙏